cmd/kops/export_kubecfg.go

```diff
@@ -92,7 +94,7 @@ func RunExportKubecfg(ctx context.Context, f *util.Factory, out io.Writer, optio
 			return fmt.Errorf("cannot use both --all flag and positional arguments")
 		}
 	}
-	if options.admin && options.user != "" {
+	if options.admin != 0 && options.user != "" {
 		return fmt.Errorf("cannot use both --admin and --user")
```

Comment:
Should we allow setting the duration when `--user` is specified?  (--user is not an option I use myself, so I don't really know)

Reply:
`--user` doesn't generate credentials, so a duration isn't relevant.